inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongGroupController.java

```diff
@@ -95,6 +97,12 @@ public Response<InlongGroupTopicInfo> getTopic(@PathVariable String groupId) {
         return Response.success(groupService.getTopic(groupId));
     }
 
+    @GetMapping(value = "/group/listTopicsByTag/{clusterTag}")
```

Comment:
Perhaps `listTopics` is better for adding other query parameters.

Reply:
You mean ` @GetMapping(value = "/group/listTopics/{clusterTag}")` ?
Maybe it will be confused with the method to list all topics by groupId?

Reply:
It is written as `@PostMapping(value = "/group/listTopics")`, and receives a request body like `/group/list`, which can facilitate the extension of parameters other than `clusterTag`, such as querying by a list of `groupId`.

If you do not modify it this way, then if you need to query the topic list according to the `groupId` or another field in the future, you need to add another API to support it.

Reply:
Got it, thx, I will fix it.